src/widget/tabs/remote-tab.js

```diff
@@ -135,17 +135,24 @@ class RemoteTab {
 
     registerMessage('open-new-window', iframe, message => {
       var interval, popup, resolve
+      
       if (this.settings.debugUploads) {
-        debug('Open new window message.', this.name)
+        log('Open new window message.', this.name)
```

Comment:
should we wrap it into 'if (process.env.NODE_ENV !== 'production')`?

Reply:
yes, missed this